README.md

```diff
@@ -76,14 +77,13 @@ Put the remaining `APP_SID` configuration info into your application server by o
 
 Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test.
 
-Open up a browser and visit the URL for your application server's Access Token endpoint: `https://{YOUR-SERVER}/accessToken`. If everything is configured correctly, you should see a long string of letters and numbers, which is a Twilio Access Token. Your Android app will use a token like this to connect to Twilio.
+Open up a browser and visit the URL for your application server's Access Token endpoint: `https://{YOUR_SERVER_URL}/accessToken`. If everything is configured correctly, you should see a long string of letters and numbers, which is a Twilio Access Token. Your Android app will use a token like this to connect to Twilio.
 
 ### <a name="bullet6"></a>6. Run the app
 
-Paste the Access Token into the VoiceActivity.java. 
-
-<img width="700px" src="images/quickstart/paste_token.png"/>
+Paste the public URL of your application server’s `https://{YOUR_SERVER_URL}/accessToken` endpoint into `TWILIO_ACCESS_TOKEN_SERVER_URL` in VoiceActivity.java. Make sure to include `/accessToken` in the URL path.
 
+<img width="600px" src="images/quickstart/paste_token.png"/>
```

Comment:
Neat picking - we are pasting a URL now, not a token. The image name should be paste_token_server_url.